src/main.js

```diff
@@ -1,89 +1,89 @@
 import http from 'http';
 import socketio from 'socket.io';
-import Monopoly from "./monopoly";
+import Monopoly from './monopoly';
 
 const server = http.createServer();
 const io = socketio(server);
 const monopolyGame = new Monopoly();
 
 io.on('connect', (socket) => {
```

Comment:
Suggest moving `monopolyGame.emitter.on('promptWhatever',)` outside of `io.on('connect')` to eventually moved into another area.  Which I describe below...

So basically in the interest of supporting multiple monopoly games running simultaneously what you'll want is to have a startGame/joinGame event that the server listens to.  This module should probably eventually end up looking like the below probably.   Let me know if this makes sense.  It should fix some of the issues you've been hitting. 

```
const monopolyGamesByID = {}; // This should be globally accessible through the module.

function setUpEvents(game) {
     game.emitter.on('promptBuy', (property, player) => {
          player.emit('prompt', stuff)
     });

     game.emitter.on('start', () => {
          game.players.forEach((player) => {
               player.emit('GameStarted', game.boardstate, game.player, etc);
          });
     })

     // etc. etc.
}


io.on('connect', (socket) => {
     const state = {currentGame: null};

     socket.on('startGame', () => {
          const game = new Monopoly(...);
          state.currentGame = game;
          game.players.push(socketOrSomeObjectContainingSocket);
          const id = userGeneratedID || randomlyGeneratedID();
          monopolyGamesByID[id] = game;

          setUpEvents(game);
      
          socket.emit('gameCreated', id);
     });

     socket.on('joinGame', (gameID) => {
          const game = monopolyGamesByID[gameID];
          if (game && !game.inProgress) {
               game.players.push(socketOrSomeObjectContainingSocket);
               state.currentGame = game;
               if (game.seatsAvailable === 0) { 
                    game.start();
               } 
          }
     });

});
```